widget.cpp

```diff
@@ -9,7 +9,8 @@
 #include <QPen>
 #include <QPainter>
 #include <QRgb>
-#include <QtConcurrent/QtConcurrent>
+#include <QtConcurrentRun>
```

Comment:
This doesn't build for me using a freshly downloaded Qt 5.8 SDK on Linux, I see it in QtConcurrent/QtConcurrentRun instead